specification/resources/resource-manager/Microsoft.Solutions/stable/2017-09-01/managedapplications.json

```diff
@@ -869,6 +869,23 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+    "/providers/Microsoft.Solutions/operations": {
+      "get": {
+        "tags": [
+          "Solutions"
+        ],
+        "operationId": "Solutions_Operations",
+        "description": "Gets all the preview solution operations.",
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array containing metadata about the solution operations. The metadata includes the name of the solution operation, the provider, the resource, the operation, and the description.",
+            "schema": {
+              "$ref": "#/definitions/OperationsList"
+            }
+          }
+        }
```

Comment:
though this API is not page-able, it's recommended to add the extension
```
        "x-ms-pageable": {
          "nextLinkName": "nextLink"
        }
```

this will instruct code-gen to avoid emitting a wrapper model for the array

refer [this](https://github.com/Azure/azure-rest-api-specs/blob/1ed16067a3dd7534f40a2996b44a5a9a0dd782f2/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json#L66)

Reply:
Gets all the solution operations